avr.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ghcr.io/modm-ext/modm-build-base:latest
+FROM ghcr.io/modm-ext/modm-build-base:TAG
 LABEL maintainer="Niklas Hauser <niklas.hauser@rwth-aachen.de>, Raphael Lehmann <raphael+docker@rleh.de"
```

Comment:
There is a missing ">" in the maintainer label, you could fix that as well if you like. Same for the RISC-V docker file.

Reply:
Done: 2490e4b